### ISSUE TITLE
how to generate a real tts

### ISSUE THREAD

**Issue by alain2208** [ttsunion/Deep-Express#5]
Hi currently your scripts re generate the original wav.
I did not find the way to do a text to speech based on a random phrase different from the original one. Maybe still not implemented?

**Comment by FonzieTree** [comment]
Hi, @alain2208, thank you for your comment. I think your issue is the same to isssue 4, please visit https://github.com/ttsunion/Deep-Expression/issues/4.

**Comment by alain2208** [comment]
yes it seems the same thanks

**Comment by surenreddy** [comment]
Why is "y:wav" is required as input for inference? 

Is it possible to generate audio for new labels(batch_size=4) that are not in the training set? In that case what should be "y:wav" ?
